css/pages/index.css

```diff
@@ -310,13 +326,14 @@ div.handles-container {
 .event-title {
   width: auto;
   font-weight: 400;
-  font-size: 2rem;
+  font-size: 2.8rem;
   text-align: center;
   color: #e30062;
 }
 
 .upcoming-event {
   font-size: 1.5rem;
+  font-family: raleway;
```

Comment:
Please remove the `font-family` attribute from here as you have already defined in `body`

Reply:
Done